doc/internal/man3/ossl_method_construct.pod

```diff
@@ -41,13 +43,25 @@ on provider dispatch tables need to do so in exactly the same way.
 ossl_method_construct() does this while leaving it to the sub-systems
 to define more precisely how the methods are created, stored, etc.
 
+It's important to keep in mind that a method is identified by three things:
```

Comment:
Typo: to keep in mind *is* that

Reply:
Uhmmmm, considering the sentence starts with "It's", I doubt that.

Reply:
Duh.